projects/demo/src/modules/components/tabs/tabs.template.html

```diff
@@ -7,10 +7,10 @@
     <ng-template pageTab>
         <p i18n>
             Component for creating tabs. It emulates appearance and behaviour of
-            native iOS and Android components when
-            <code>TUI_MOBILE_AWARE</code>
-            token is
-            <code>true</code>
+            native iOS and Android components with
+            <code>tuiMobileTabs</code>
+            directive from
+            <code>@taiga-ui/addon-mobile</code>
```

Comment:
I think that many developers will forget about importing `TuiMobileTabsModule`.

I think we can write about it here.